src/MacVim/MMCoreTextView.m

```diff
@@ -626,7 +669,7 @@ - (BOOL)mouseDownCanMoveWindow
 
 - (BOOL)isOpaque
 {
-    return YES;
+    return self.layer == nil || self.defaultBackgroundColor.alphaComponent == 1;
```

Comment:
I think this is currently breaking the `transparency` setting. Just do `set transparency=50` and compare the screenshots. If you use colorscheme `industry` (which has a black background) and overlay it on a white background, and you screencap it, the color should be exactly 0.5 / 128. With this change I think there is some double blending going on.

Also, in pre-Mojave builds (<=10.13), transparency just stopped working it seems. The background no longer shows through.

The current transparency support is a little brittle and has different behaviors under Mojave due to the different renderer used (see #805), so you may want to double-check that.

Reply:
Fixed the double blending in e5c2fba9542161d25fe1fd74118e409fea0fb69a. I don't have access to a 10.13 machine to test right now — I may try to set up a VM, but for the moment it's safest to assume that I only have access to 10.14 and 10.15, unfortunately.